database/migrations/2022_10_29_143858_create_product_items_table.php

```diff
@@ -22,10 +22,13 @@ public function up()
             $table->enum('size', [
                 'S',
                 'M',
-                'L'
+                'L',
+                'ONE_SIZE'
```

Comment:
Lo de ONE_SIZE es el texto que se va a guardar en la base de datos y lo que se devuelve en JSON al hacer una petición, yo pondria `Única` para poder sacarlo en el cliente tal cual está en el JSON y en el enum pondria `case ONE_SIZE = "Única"`

Reply:
Modificado.

Reply:
Lo has cambiado en el enum pero hay que cambiarlo aquí también, el texto del enum tiene que ser igual que lo que se guarde en la base de datos.
```suggestion
                'Única'
```